recipes/triqs_tprf/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "2.2.0" %}
+
+package:
+  name: triqs_tprf
+  version: {{ version }}
+
+source:
+  url: https://github.com/TRIQS/tprf/releases/download/{{ version }}/tprf-{{ version }}.tar.gz
+  sha256: 87389c71bdf690e05b72a5cd7dd2f4f7d8a8ac69b15f1481de1960e7740efc9f
+
+build:
+  number: 0
+  skip: True  # [win or py>30]
+  merge_build_host: True
+
+requirements:
+  build:
+    - cmake
+    - make
+    - clangxx >=8  # [osx]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - triqs 2.2
+  run:
+    - triqs 2.2
+
+test:
+  imports:
+    - triqs_tprf
+
+about:
+  home: https://triqs.github.io/tprf
+  license: GPLv3
+  license_file: LICENSE.txt
```

Comment:
Please add a license_family entry.  CC @conda-forge/core This would have been caught by https://github.com/conda-forge/conda-smithy/pull/1256

Reply:
Done